src/App.jsx

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
+import { Sum } from './components/Sum';
+
 import './App.css';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
```

Comment:
```suggestion
    <Sum a={2} b={3} />
```
You can make it in one line as there are fewer than 3 attributes and they don't make your line too wide